teuthology/provision/fog.py

```diff
@@ -227,8 +227,8 @@ def schedule_deploy_task(self, host_id):
         for task in host_tasks:
             timestamp = task['createdTime']
             time_delta = (
-                datetime.utcnow() - datetime.strptime(
-                    timestamp, self.timestamp_format)
+                datetime.datetime.now(datetime.timezone.utc) - datetime.datetime.strptime(
+                    timestamp, self.timestamp_format).replace(tzinfo=datetime.timezone.utc)
```

Comment:
Maybe... maybe we can just split this into several lines:
```
created = datetime.datetime.strptime(timestamp, self.timestamp_format)
utc_now = datetime.datetime.now(datetime.timezone.utc)
utc_created = created.replace(tzinfo=datetime.timezone.utc)
time_delta = (utc_now - utc_created).total_seconds()
```

Reply:
but I'm wondering, it would be way better if python evangelists create three or for incapsulated namespaces, so we can graciously use: `datetime.datetime.datetime.datetime.timezone.utc`

Reply:
It's a little ridiculous, yeah. For testing purposes I felt the need to import the package, but it would be nice to split it over multiple lines. Still, I don't want to hold up the review so I'll do a follow-up at some point. Thanks!